terraform/openstack-modules.sample.tf

```diff
@@ -14,6 +15,9 @@ variable floating_ip_pool { default = "name-of-your-floating-ip-pool" }
 # Run 'nova image-list' to get your image name
 variable image_name  { default = "your-CentOS-7" }
 
+# DNS servers passed to Openstack subnet
+variable dns_nameservers { default = "" } # comma separated list of ips, e.g. "8.8.8.8,8.8.4.4"
```

Comment:
Yeah, I use a `compact(split(...))` to turn the empty string into an empty array, which will treat dns_nameservers as if it was not set at all.